### PR TITLE
Feature/breadcrumb chapter numbering

### PR DESCRIPTION
**Pull request by pandafulmanda**
screen shot 2015-05-27 at 2 38 56 pm
screen shot 2015-05-27 at 2 41 01 pm

### CODE REVIEW

File: resources/styles/components/task/breadcrumbs.less [hunk 1]
```diff
@@ -28,39 +31,62 @@
...
-      color: @tutor-incorrect-color;
+    i:not(.icon-incorrect):not(.icon-correct) {
+      display: none;
     }
   }
-}
\ No newline at end of file
+}
+
+.steps {
```
Comment (philschatz) [inline resources/styles/components/task/breadcrumbs.less]: Can these be scoped inside something like `.task-reading`? (Or enumerate all the possible containers)

File: src/components/task/breadcrumbs.cjsx [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+  mixins: [ChapterSectionMixin, CrumbMixin]
 
   propTypes:
     id: React.PropTypes.string.isRequired
     currentStep: React.PropTypes.number.isRequired
     goToStep: React.PropTypes.func.isRequired
 
+  componentWillReceiveProps: ->
+    crumbs = @getCrumableCrumbs()
+    TaskStepStore.setMaxListeners(3 * crumbs.length) if crumbs?
```
Comment (philschatz) [inline src/components/task/breadcrumbs.cjsx]: Can you include a comment as to why 3 is chosen?

File: src/components/task/crumb-mixin.cjsx [hunk 3]
```diff
@@ -30,6 +30,17 @@ module.exports =
 
     doesAllowSeeAhead or index <= latestIndex
 
+  _buildSectionLabel: (chapter_section, crumbs) ->
+    chapterSection = _.clone(chapter_section)
+
+    # ignore 0 in chapter sections
```
Comment (philschatz) [inline src/components/task/crumb-mixin.cjsx]: Should `@sectionFormat` (or `<ChapterSection>` component) do this?

Comment (pandafulmanda) [inline src/components/task/crumb-mixin.cjsx]: where the 0 is ignored..? I am unsure..seems like that should be the case everywhere right?  who should I ask?

Comment (jpslav) [inline src/components/task/crumb-mixin.cjsx]: Ask @jason-holmes for issues regarding how the 0 should be treated.

**Pull request merged**